assets/js/modules/index.test.js

```diff
@@ -49,11 +49,17 @@ function getComponentNames( componentPath ) {
 describe( 'all modules', () => {
 	describe.each( directories( '.' ) )( '%s', ( moduleSlug ) => {
 		const components = directories( `${ moduleSlug }/components` );
-		if ( ! components.length ) {
+
+		// Filter out the custom-dimensions-report-options directory
+		const filteredComponents = components.filter(
```

Comment:
Hey @zutigrm, are these changes needed or could they have been introduced by accident? When I remove them and run the test suite locally, it still passes...

Reply:
Hey @techanvil, they are not now, as the folder structure is removed in favour of date/report options functions being in the same file. I will remove that from the test in my next issue

Reply:
Sounds good, thanks @zutigrm!